### PR TITLE
Feature/wizard steps123

### LINKED ISSUE #31: Display projects considering active user's sharing access
Basically, it is about using this call:

http://dev.eyeseetea.com:8080/api/categoryOptions.json?filter=categories.id:ilike:MRwzyV0kXv9 

instead of:

http://dev.eyeseetea.com:8080/api/26/categories.json?filter=id:eq:MRwzyV0kXv9&fields=id,%20categoryOptions%5Bid,%20code,displayName,startDate,endDate,categories[id],organisationUnits%5B*%5D%5D&paging=false

### PR DESCRIPTION
**Pull request by tokland**
Closes #31, #34, #29, #30, #28, #31, #30,  #32, #29,

### CODE REVIEW

**Comment by adrianq** [comment]
I will wait for #31 and #33 before reviewing and merging

File: README.md [hunk 1]
```diff
@@ -13,9 +13,10 @@ $ yarn start
```
Comment (adrianq) [inline README.md]: I haven't managed to make it work without d2-ui link. Anyway let's try to merge this PR and we can review the error together during our next meeting...

File: yarn.lock [hunk 2]
```diff
@@ -1492,9 +1514,9 @@ d2-manifest@^1.0.0:
     minimist "^1.1.0"
     readline-sync "^1.4.1"
 
-d2-ui@26.1.7:
+"d2-ui@github:EyeSeeTea/d2-ui#feature/installable-from-github":
```
Comment (tokland) [inline yarn.lock]: Right, it should be `#dataset-configuration`

**Comment by tokland** [comment]
All comments applied.

**Pull request merged**